clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -18,52 +18,6 @@
 import defaults as d
 
 
-def api_call_latency(title, verb, scope, threshold):
-    return d.Graph(
-        title=title,
-        targets=[
-            g.Target(expr=str(threshold), legendFormat="threshold"),
-            g.Target(
-                expr='apiserver:apiserver_request_latency_1m:histogram_quantile{quantile="0.99", verb=~"%(verb)s", scope=~"%(scope)s"}'
-                % {"verb": verb, "scope": scope},
-                legendFormat="{{verb}} {{scope}}/{{resource}}",
-            ),
-        ],
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
-    )
-
-
-CLUSTERLOADER_PANELS = [
-    d.simple_graph(
```

Comment:
Sorry, my bad. I meant to delete only this graph (requests), the other graphs (api-call-latency graphs) are actually useful and don't depend on the client label.

Reply:
No problem, `CLUSTERLOADER_PANELS` is back with only first graph deleted.